gtsam/navigation/tests/testGPSFactor.cpp

```diff
@@ -71,7 +70,7 @@ TEST( GPSFactor, Constructor ) {
   EXPECT(assert_equal(Z_3x1,factor.evaluateError(T),1e-5));
 
   // Calculate numerical derivatives
-  Matrix expectedH = numericalDerivative11<Vector,Pose3>(
+  Matrix expectedH = numericalDerivative11<Vector, Pose3>(
```

Comment:
Why changes in unrelated files? Maybe merge in develop again? Or move these to new PR

Reply:
Merged in latest `develop`.